standards/github_folder_template.md

```diff
@@ -0,0 +1,50 @@
+# The .github folder and it's functionality
+
+### What is the .github folder?
+
+The `.github` folder allows users to store GitHub related files/folders. Some of these files and folders perform specials functions for organizing your repo, issues, and PRs.
+
+Here are some of those functions:
+- Creating issue templates (ISSUE_TEMPLATE folder)
```

Comment:
Can we link this to the issue template file inside the standards and practices? So when some is reading the document they can click on it and it takes them to the template?